### DISCUSSION TITLE
[Script request] InspIRCd

### DISCUSSION THREAD

**Discussion by Zignet64** [community-scripts/ProxmoxVE#456] [Request script] [answered]
### Application Name

InspIRCd

### Website

https://www.inspircd.org/

### Description

Would be nice to have an LXC server ircd ! :) Thanks !

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by quantumryuu** [comment]
Assign me please! Already pushed a PR 😃

**Reply by Zignet64** [reply]
The PR : [The Lounge](https://github.com/community-scripts/ProxmoxVE/pull/571), is not a server Inspircd

**Reply by MickLesk** [reply]
He has made 3 PRs i think ^^

**Reply by quantumryuu** [reply, marked as answer]
Hello @Zignet64!
This is the PR you're looking for #576 😃

**Reply by Zignet64** [reply]
Thanks !!! :)
